tests/model_serving/model_runtime/mlserver/utils.py

```
@@ -0,0 +1,316 @@
+                send_rest_request(url=f"{host}{rest_endpoint}", input_data=input_data, verify=False)
+                if is_rest
+                else send_grpc_request(url=host, input_data=input_data, root_dir=root_dir, insecure=False)
+            )
+
+    elif deployment_mode == KServeDeploymentType.SERVERLESS:
+        base_url = isvc.instance.status.url.rstrip("/")
+        if is_rest:
+            return send_rest_request(url=f"{base_url}{rest_endpoint}", input_data=input_data, verify=False)
+        else:
+            grpc_url = get_grpc_url(base_url=base_url, port=MLSERVER_GRPC_REMOTE_PORT)
+            return send_grpc_request(url=grpc_url, input_data=input_data, root_dir=root_dir, insecure=True)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)